pkg/policy/api/selector.go

```diff
@@ -32,6 +32,7 @@ var log = logging.DefaultLogger
 // EndpointSelector is a wrapper for k8s LabelSelector.
 type EndpointSelector struct {
 	*metav1.LabelSelector
+	k8sLbls.Selector
```

Comment:
I would have hidden that into a private field, instead of using composition, e.g.:
```
k8sSelector k8sLbls.Selector
```
That would be better encapsulated.